test/test_containers.py

```diff
@@ -16,6 +16,7 @@ def ss_container():
     ss.attrs["test_attr1"] = "hello"
     ss.vis.attrs["test_attr2"] = "hello2"
     ss.weight.attrs["test_attr3"] = "hello3"
+    ss.index_attrs["freq"]["alignment"] = 1
```

Comment:
Some of these tests must have been accidentally removed because this shouldn't work